deployment/ansible/roles/nih-wayfinding.nodejs/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Determine what version of NPM is installed
+  command: npm --version
+  register: npm_version
+
+- name: Upgrade NPM
```

Comment:
This probably needs to go into the `azavea.nodejs` role at some point. The version of NPM the comes bundled with Node.js doesn't work well.